sim/hunter/hunter.go

```diff
@@ -79,53 +92,50 @@ func (hunter *Hunter) GetHunter() *Hunter {
 }
 
 func (hunter *Hunter) AddRaidBuffs(raidBuffs *proto.RaidBuffs) {
-	// if hunter.Talents.TrueshotAura {
-	// 	raidBuffs.TrueshotAura = true
-	// }
-	// if hunter.Talents.FerociousInspiration == 3 && hunter.pet != nil {
-	// 	raidBuffs.FerociousInspiration = true
-	// }
+	if hunter.Talents.TrueshotAura {
+		raidBuffs.TrueshotAura = true
+	}
+	if hunter.Talents.FerociousInspiration && hunter.Pet != nil {
+		raidBuffs.FerociousInspiration = true
+	}
 }
 func (hunter *Hunter) AddPartyBuffs(_ *proto.PartyBuffs) {
 }
+func (hunter *Hunter) CalculateMasteryPoints() float64 {
+	return hunter.GetStat(stats.Mastery) / core.MasteryRatingPerMasteryPoint
+}
```

Comment:
why is there mastery related stuff in the core hunter?

Reply:
So I can use it more than once in the upper parts. I can remove it np though

Reply:
Ah this then could just be moved to the core in character if its not hunter related but a shorthand for getting the current total points

Reply:
Moved out to core